data/json/items/tools.json

```diff
@@ -8143,6 +8143,21 @@
     "flags": ["HYGROMETER"]
   },
   {
+    "id": "hydrometer",
+    "type": "TOOL",
+    "symbol": ";",
+    "color": "light_blue",
+    "name": "hydrometer",
+    "name_plural": "hydrometers",
+    "description": "An instrument that measures the specific gravity (relative density) of liquids-the ratio of the density of the liquid to the density of water.  It make the production of maple syrup easier and more efficient.",
```

Comment:
"It make"

Also, don't specify what does the item do exactly, that makes it way too specific.